bin/sync

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+from cookiecutter.main import cookiecutter
+import git
+import json
+import os
+import requests
+from requests.auth import HTTPBasicAuth
+import shutil
+import sys
+import subprocess
+import tempfile
+import syncutils.template
+
+# Set the default nf-core pipeline template branch
+DEF_TEMPLATE_BRANCH = "TEMPLATE"
+# The GitHub base url or the nf-core project
+GH_BASE_URL = "https://{token}@github.com/nf-core/{pipeline}"
+# The current cookiecutter template url for nf-core pipelines
+NF_CORE_TEMPLATE = os.path.join(
```

Comment:
You don't need to define this any more if you're using the function from the `nf_core` package.